Simplenote/Classes/SPTagTableViewCell.swift

```diff
@@ -0,0 +1,90 @@
+import Foundation
+import UIKit
+
+
+// MARK: - SPTagTableViewCell
+//
+@objcMembers
+class SPTagTableViewCell: UITableViewCell {
+
+    /// Note's Title
+    ///
+    var titleText: String? {
+        get {
+            textLabel?.text
+        }
+        set {
+            textLabel?.text = newValue
+        }
+    }
+
+
+    // MARK: - Overridden Methods
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        refreshStyle()
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        refreshStyle()
+    }
+}
+
+
+private extension SPTagTableViewCell {
+
+    /// Refreshes the current Style current style
+    ///
+    func refreshStyle() {
+        textLabel?.textColor = Style.textColor
+        backgroundColor = Style.backgroundColor
+
+        let selectedView = UIView(frame: bounds)
+        selectedView.backgroundColor = Style.selectionColor
+        selectedBackgroundView = selectedView
+    }
+}
+
+
+// MARK: - SPTagTableViewCell
+//
+extension SPTagTableViewCell {
+
+    /// Returns the Height that the receiver would require to be rendered
+    ///
+    /// Note: Why these calculations? why not Autosizing cells?. Well... Performance.
+    ///
+    static var cellHeight: CGFloat {
+        let verticalPadding: CGFloat = 12
```

Comment:
I suppose there isn't much use in declaring a constant for this value, eh?

Reply:
Good call! Moving over to the `Style` enum!